st2api/st2api/controllers/v1/pack_configs.py

```diff
@@ -107,6 +107,8 @@ def put(self, pack_config_content, pack_ref, requester_user, show_secrets=False)
             config_api.validate(validate_against_schema=True)
         except jsonschema.ValidationError as e:
             raise ValueValidationException(six.text_type(e))
+        except ValueValidationException as e:
+            raise ValueValidationException(six.text_type(e))
```

Comment:
Can you add a test case for this try-except block?

Reply:
Are you still working on this?

Reply:
Yes. Need to start looking at existing tests in test_pack_configs and see how to address this.

Reply:
Done